counting-svg/counting-svg.sol

```diff
@@ -0,0 +1,101 @@
+
+        string memory tailSVG = "</svg>";
+        string memory bodySVG = string(
+            abi.encodePacked(
```

Comment:
If you want, you can use `string.concat` as of 0.8.12 of solidity https://ethereum.stackexchange.com/questions/729/how-to-concatenate-strings-in-solidity

Optional

Reply:
Seems that `abi.encodePacked` is more efficient.
https://ethereum.stackexchange.com/a/56337/94286